bindings/2.200/GeometryDash.bro

```diff
@@ -2884,7 +2884,7 @@ class SliderThumb : cocos2d::CCMenuItemImage {
 	~SliderThumb();
 
 	// inlined on windows
-	float getValue() {
+	float getValue() = mac 0x2e02f0 {
```

Comment:
sir that is not valid syntax

Reply:
i know, thats why the action is failing. so what is the correct syntax?

Reply:
just `float getValue() = mac 0x2e02f0;`

Reply:
its inlined on windows so i'll just effectively be making it unusable on windows, which is a no no

Reply:
dude windows isnt on 2.200

Reply:
just keep the inlined implementation tbh unless youre hooking the func ig

Reply:
> dude windows isnt on 2.200
> 

it was

Reply:
> just keep the inlined implementation tbh unless youre hooking the func ig

i'll remove the mac binding ig